### DISCUSSION TITLE
Firefly saldo 0

### DISCUSSION THREAD

**Discussion by jjjjjjjjjj20051** [iotaledger/firefly#2394] [Help & Support]
Hello, I have a problem, I had to restore firefly with the 24 words, I agree to recover the account, all the transactions, it is more in the tangle appear, but the balance comes out O, in fact I am calm because the last address I did the staking with all the iotas appears the transaction, and that address somehow has the balance, which belongs to my seed. Another thing is that the tangle does not show balance. Thank you very much

**Comment by MiKeF05** [comment]
If the staked address has the correct balance when you check it with the explorer then it is most likely a visual bug in Firefly. Can you try the balance finder in the settings? Do you use a ledger device?
